components/function-controller/cmd/jobinit/main.go

```diff
@@ -34,7 +34,11 @@ func main() {
 	log.Printf("Clone repo from url: %s and commit: %s...\n", cfg.RepositoryUrl, cfg.RepositoryCommit)
 	commit, err := operator.Clone(cfg.MountPath, gitOptions)
 	if err != nil {
-		log.Fatalln(errors.Wrapf(err, "while cloning repository: %s, from commit: %s", cfg.RepositoryUrl, cfg.RepositoryCommit))
+		if git.IsAuthErr(err) {
+			log.Printf("while cloning repository bad credentials were provided, errMsg: %s", err.Error())
```

Comment:
suggestion only 😄 
![alt text](https://static.wikia.nocookie.net/starwars/images/d/d6/Yoda_SWSB.png/revision/latest?cb=20150206140125)

```suggestion
			log.Printf("bad credentials provided while cloning the repository, errMsg: %s", err.Error())
```